packages/jekyll-format/jekyll-format.0.2.0/opam

```diff
@@ -18,7 +18,7 @@ depends: [
   "rresult"
   "ptime"
   "fpath"
-  "yaml"
+  "yaml" {>= "2.0.0"}
```

Comment:
This was only required for the tests, it was perfectly fine before without any constraints. This should've been:
```
  "yaml"
  "yaml" {with-test & >= "2.0.0"}
```